docs/theme/default.rst

```diff
@@ -182,65 +182,23 @@ default order if the user does not configure a different template.
    If running in a terminal multiplexer, will be colored. See
    :attr:`LP_COLOR_IN_MULTIPLEXER`.
 
+.. attribute:: LP_PROXY
+
+   A ↥ (:attr:`LP_MARK_PROXY`) if an HTTP proxy is in use. See
+   :attr:`LP_ENABLE_PROXY`.
+
 .. attribute:: LP_SHLVL
 
    The number of nested shells, prefixed with :attr:`LP_MARK_SHLVL`, all colored
    with :attr:`LP_COLOR_SHLVL`. Can be disabled by attr:`LP_ENABLE_SHLVL`.
 
    .. versionadded:: 2.1
 
-.. attribute:: LP_SCLS
-
-   The current Red Hat Software Collections environment. See
-   :attr:`LP_ENABLE_SCLS`.
-
-.. attribute:: LP_AWS_PROFILE
-
-   The current active AWS Profile. See
-   :attr:`LP_ENABLE_AWS_PROFILE`.
-
-   .. versionadded:: 2.1
-
-.. attribute:: LP_CONTAINER
-
-   The container status for the current shell. See :attr:`LP_ENABLE_CONTAINER`.
-
-   .. versionadded:: 2.1
-
-.. attribute:: LP_VENV
+.. attribute:: LP_DEV_ENV
 
-   The current Python (or Conda) virtual environment. See
-   :attr:`LP_ENABLE_VIRTUALENV`.
-
-.. attribute:: LP_NODE_VENV
+   Sections related to development environments (see section `dev_env`_ below).
```

Comment:
Either here or in the linked Dev Env section, there needs to be a note explaining that `LP_DEV_ENV` is the new way to do the same thing that all the others did before, and how if a user doesn't like the order they can use the old sections manually.